config/locales/de.yml

```diff
@@ -172,3 +172,17 @@ de:
       previous: "<"
       next: ">"
       truncate: "..."
+  email_auth:
+    header: "Mit E-Mail anmelden"
+    enter_email: "Bitte geben Sie Ihre E-Mail-Adresse ein"
+    submit: "Autorisieren"
+    email_sent: "Eine E-Mail wurde an %{email} mit Details zum Einloggen gesendet"
+    invalid_email: "Bitte geben Sie eine gültige E-Mail-Adresse ein"
```

Comment:
we do not use "Sie" but "Du" in the rest of our communication

Reply:
I speak no German (nor Polish) I'm afraid. I asked ChatGPT to do the translations hoping they would be a good starting point that could be improved by native speakers. Still, I asked again to rewrite the German texts in the colloquial form. Hoping they're not too bad!!